src/Servers/HttpSys/src/RequestProcessing/Request.cs

```diff
@@ -20,6 +20,9 @@ namespace Microsoft.AspNetCore.Server.HttpSys
 {
     internal sealed class Request
     {
+        private static readonly Action<ILogger, Exception?> _errorInReadingCertificate =
```

Comment:
Keep the static log class just make it a private class in this class.

Reply:
I wasn't clear on that, sorry...

Reply:
No problem!